README.md

```diff
@@ -30,6 +44,58 @@ to identify in the source code (as LibCST will find all matches of this construc
 
 ## Getting Started
 
+### Importing into Another Grading Tool
+
+1. Install from PyPi by running `poetry add pypicount` in your terminal.
+2. Import the PyPiCounter package withe the syntax: `import pypicount`.
+3. Run the functions in the package using the `PiPyCount.[function]` syntax.
+For example, `PiPyCount.count_class_definitions(file_name)`
+
+The list of functions available are:
+
+```python
+
+count_class_definitions(file_name)                           
+# returns the number of class definitions
+
+count_comments(file_name)                                    
+# returns the number of comments
+
+count_import_statements(file_name)                           
+# returns the number of import statements
+
+count_for_loops(file_name)                                   
+# returns the number of for loops
+
+count_while_loops(file_name)                                 
+# returns the number of while loops
+
+count_function_definitions(file_name)                        
+# returns the number of function definitions  
+
+count_functions_without_docstrings(file_name)                
+# returns the number of functions without docstrings
+
+count_functions_with_docstrings(file_name)                   
+# returns the number of functions with docstrings
+
+count_classes_with_docstrings(file_name)                     
+# returns the number of classes with docstrings
+
+count_classes_without_docstrings(file_name)                  
+# returns the number of classes without docstrings
+
+count_function_parameters(file_name, function_name)          
+# returns the number of function parameters
+# after specifying the function name
+
+count_assignment_statements(file_name)                       
+# returns the number of assignment statements
+
+```
+
+### Stand-Alone Usage
+
 Users can get started with this project by following the following steps:
 
 1. Clone this repository and `cd` into the project folder
```

Comment:
Instead of requesting that users clone the repository, since your tool is now on PyPI they can install it with `pipx` or `pip`, which you should recommend. You can move the `poetry install`-related suggestions to a "contributing" section, or remove them altogether.